packages/admin/cms-admin/src/redirects/RedirectsTable.tsx

```diff
@@ -107,6 +108,7 @@ export function RedirectsTable({ linkBlock, scope }: Props): JSX.Element {
             renderCell: (params) => <div>{params.value}</div>,
             sortable: false,
             flex: 1,
+            minWidth: 400,
             filterable: false,
         },
         {
```

Comment:
I just checked on the mobile screen and assigning `flex: 1` to `generationType`, `active` and `actions` squeezes them on mobile (the field is smaller than its content). So you could remove the `flex` prop entirely for these columns. Then they use as much space as their content needs.